docs/helm-deployment-aws_cloud.md

```diff
@@ -0,0 +1,410 @@
+--version 0.14.0 \
+--set controller.scope.enabled=true \
+--set controller.scope.namespace=$DESIREDNAMESPACE \
+--set rbac.create=true \
+-f ingressvalues.yaml \
+--namespace $DESIREDNAMESPACE
+```
+
+<details><summary>Click here to see example `ingressvalues.yaml`</summary>
+<p>
+
+```bash
+cat <<EOF > ingressvalues.yaml
+controller:
+  config:
+    ssl-redirect: "false"
+  scope:
+    enabled: true
+    namespace: $DESIREDNAMESPACE
+  publishService:
+    enabled: true
+  service:
+    targetPorts:
+      https: 80
+    annotations:
+      service.beta.kubernetes.io/aws-load-balancer-ssl-cert: ""
+      service.beta.kubernetes.io/aws-load-balancer-ssl-ports: https
+      external-dns.alpha.kubernetes.io/hostname: $DESIREDNAMESPACE.YourDNSZone
+      service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy: "ELBSecurityPolicy-TLS-1-2-2017-01"
+EOF
+```
+
+</p>
+</details>
+
+Option 2:
+
+```bash
+# Helm install nginx-ingress along with args
+export AWS_CERT_ARN="arn:aws:acm:<AWS-REGION>:<AWS-AccountID>:certificate/<CertificateId>"
+export AWS_CERT_POLICY="ELBSecurityPolicy-TLS-1-2-2017-01"
+
+helm install stable/nginx-ingress \
+--version 0.14.0 \
+--set controller.scope.enabled=true \
+--set controller.scope.namespace=$DESIREDNAMESPACE \
+--set rbac.create=true \
+--set controller.config."force-ssl-redirect"=\"true\" \
+--set controller.config."proxy-body-size"="100m" \
+--set controller.service.targetPorts.https=80 \
+--set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-backend-protocol"="http" \
+--set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-ssl-ports"="https" \
+--set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-ssl-cert"=$AWS_CERT_ARN \
+--set controller.service.annotations."external-dns\.alpha\.kubernetes\.io/hostname"="$DESIREDNAMESPACE.dev.alfresco.me" \
+--set controller.service.annotations."service\.beta\.kubernetes\.io/aws-load-balancer-ssl-negotiation-policy"=$AWS_CERT_POLICY \
+--set controller.publishService.enabled=true \
+--namespace $DESIREDNAMESPACE
+```
+Adjust the above values accordingly (ex: `--version` of `nginx-ingress`, `controller.service.annotations` etc.)
+
+* To view the AWS ELB DNS name, note down the `nginx-ingress` release from the output of the above command.  It should be something like (Ex: `NAME:  knobby-wolf`)
+```bash
+export INGRESSRELEASE=knobby-wolf
+export ELBADDRESS=$(kubectl get services $INGRESSRELEASE-nginx-ingress-controller --namespace=$DESIREDNAMESPACE -o jsonpath={.status.loadBalancer.ingress[0].hostname})
+echo $ELBADDRESS
+```
+
+* Create an EFS storage on AWS and make sure it's in the same VPC as your cluster by following this [guide](https://github.com/Alfresco/alfresco-dbp-deployment#6-efs-storage-note-only-for-aws).  Make sure you open inbound traffic in the security group to allow NFS traffic.  Below is an example of how to create it with the AWS Cli.
```

Comment:
Should we mention S3 Connector (and provide link to docs) as an alternative add-on option for ACS Content Repository ?